vimrc

```diff
@@ -195,7 +195,9 @@ let g:spec_runner_dispatcher = 'call Send_to_Tmux("clear\n{command}\n")'
 let g:indentobject_meaningful_indentation = ['haml', 'sass', 'yaml', 'markdown']
 let g:indentLine_fileType = ['yaml']
 
-let g:rspec_command = 'call Send_to_Tmux("rspec {spec}\n")'
+if !system("tmate show-messages | grep tmate &> /dev/null")
```

Comment:
Could we also add a condition to be sure we're in tmux?  `if exists('$TMUX')` for example?